src/components/Helpers/Button/Button.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from "react";
+import css from "./Button.module.scss";
+
+interface ButtonProps {
+  color?: string;
+  disabled?: boolean;
+  children: ReactElement;
+  onClick?: any;
+}
+
+export const ButtonIcon = ({ color = "", disabled, children, onClick }: ButtonProps) => {
```

Comment:
I would have written that one differently, something like

```
export const ButtonIcon = ({className, ...props} : ButtonProps) => {

  return (
    <ButtonSolid className={`${className} ${css.icon}`} {...props}>
      {children}
    </ButtonSolid>
  );
}
```
and need to add className for ButtonSolid

I'm fine with current implementation just sharing :)

Reply:
ok updated